Jenkinsfile

```diff
@@ -43,6 +26,12 @@ def cmake_build(compiler, flags, env4make, extradebugflags, prefixpath){
     if (prefixpath != "/usr/local") {
         configargs = "-DCMAKE_PREFIX_PATH=${prefixpath}"
     }
+    
+    if(flags.contains('-DBUILD_DEV=Off'))
+    {
+    	config_targets = 'install ' + config_targets
+    	flags = '-DCMAKE_INSTALL_PREFIX=../install ' + flags
```

Comment:
Why aren't we using the default install prefix? The packages we build should be able to be used by others.

Reply:
I tried once -- it didn't work and I do not know why. I can try once more , if you wish.

Reply:
This is noted in the the PR description.

Reply:
Ah yea it may not work because it may need to be sudo. Why are we adding an install step here? When we call `make package` it will run install into the build directory, but the package will use the correct install prefix.

Reply:
I would like to test "installed" MIOpen. Some aspects of its behavior differ for DEV builds, e.g. binary cache is enabled. This may accelerate the tests. And recently I fiddled around with an issue which is not reproducible with DEV builds (#798).

Or you mean that install step is not necessary to test non-DEV build? IIRC at least installation of system databases is required.